framework/src/auxkernels/ParsedAux.C

```diff
@@ -57,10 +57,18 @@ ParsedAux::ParsedAux(const InputParameters & parameters)
   // coupled field variables
   if (isCoupled("args"))
     for (std::size_t i = 0; i < _nargs; ++i)
-      variables += (i == 0 ? "" : ",") + getFieldVar("args", i)->name();
+    {
+      auto * const field_var = getFieldVar("args", i);
+      field_var->requireQpComputations();
```

Comment:
what is happening now? are we not pre-computing the values?

Reply:
not by default for INSFV/PINSFV/WCNSFV